### PR TITLE
Updating configuration to allow CentOps integration to work for MockBot and MockClassifier

### PR DESCRIPTION
**Pull request by guypritchard**
- I've left the previous configuration in there for now as I suppose this pipeline will still run for other commits - so in theory this should work for both the CentOps integration and without.

### CODE REVIEW

File: src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public CentOpsFixture(IConfiguration configuration)
                 Host = "http://bot1/dmr-api/messages",
                 Type = "Chatbot",
                 Status = "Active",
-                ApiKey = "thisisareallylongkey"
+                ApiKey = "thisisareallylongkeyformockbot1"
```
Comment (decodingahmed) [inline src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs:69]: I can see in the docker compose file that you've added keys for both mockbots, do you also need a key for mockbot2 here in the code?

**Review by decodingahmed — APPROVED** [review]

File: src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs (hunk 1 above)
Comment (guypritchard) [reply src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs:69]: The test doesn't use bot2 currently - but I figured I'd ensure the API key was unique as I was in there - does is raise more questions than it answers?  Happy to revert based on the principle of least surprise.

**Review by martinkearn — APPROVED** [review]
This looks good to me ... presumably you have tested this locally as per the instructions in the readme? Do those instruction still hold true?

File: src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs (hunk 1 above)
Comment (decodingahmed) [reply src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs:69]: I keep forgetting that the scenario involving mockbot2 has not yet been lit up in the tests.  The person writing that scenario can add in the code for mockbot2 here.

Comment (guypritchard) [reply src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs:69]: We talked about this - bot2 will come into play in future tests, but for now it's there as a placeholder.

**Pull request merged**